CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 - Fixed a bug where the version of a dependency for Python 2.7 usage was incorrectly specified.
 
+## 1.5.3 - 2017-05-18
+### Changed
+- Changed criteria in ``civis.io.file_to_civis`` for streaming a file to include a check on if the file object is seekable.
```

Comment:
The thing that users care about is that they can now provide file-like objects which aren't seekable, right? We used to error on that. I'd rephrase this entry to make that clear.